src/components/Search/Search.css

```diff
@@ -22,7 +22,7 @@
 .search--button {
   position: absolute;
   right: 3%;
-  height:20px;
+  height: 35px;
```

Comment:
why did this get a size increase? I'm now seeing this icon clipping out of the search bar.

Reply:
it was not showing up in my bar, so i increased to see it. Must be different on your screen possibly. Ill lower it on my end

Reply:
reset

height: 20px;
width: 20px;

Reply:
![Screen Shot 2023-07-30 at 11 22 19 AM](https://github.com/mbenfowler/sound-stash/assets/76410246/5ec42892-1b2d-4eae-90af-9b55a691a85a)